grammar.js

```diff
@@ -1702,6 +1703,14 @@ module.exports = grammar({
       $.constraint,
     ),
 
+    drop_constraint: $ => seq(
```

Comment:
there is a _drop_statement further down. I think it would be better there. and not in alter